pybamm/models/full_battery_models/lithium_ion/basic_dfn.py

```diff
@@ -0,0 +1,270 @@
+#
+# Basic Doyle-Fuller-Newman (DFN) Model
+#
+import pybamm
+from .base_lithium_ion_model import BaseModel
+
+
+class BasicDFN(BaseModel):
+    """Doyle-Fuller-Newman (DFN) model of a lithium-ion battery, from [2]_.
+
+    This class differs from the :class:`pybamm.lithium_ion.DFN` model class in that it
+    shows the whole model in a single class. This comes at the cost of flexibility, and
+    in general the main DFN class should be used instead.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the model.
+
+    References
+    ----------
+    .. [2] SG Marquis, V Sulzer, R Timms, CP Please and SJ Chapman. “An asymptotic
+           derivation of a single particle model with electrolyte”. In: arXiv preprint
+           arXiv:1905.12553 (2019).
+
+
+    **Extends:** :class:`pybamm.lithium_ion.BaseModel`
```

Comment:
I got a bit confused until I realized that there are 2 different `BaseModel` classes, and that the second one overwrites the first!